spa_client_concerns.md

```diff
@@ -0,0 +1,29 @@
+# Let's attack your single page app
+
+So, you want to build a "modern" web application. Most likely, this means a "single page app" - a type of web site that relies heavily on JavaScript execution within the browser. You'll probably be using one of the many popular JavaScript frameworks to help you - something like React, Angular, Vue, etc...
```

Comment:
_I_ was suggested at some point by our editor to use "app" or "application" (consistently). Just sharing. I, personally, see a value in using both when applied to web _applications_ and mobile _apps_.